core/src/main/res/values/strings.xml

```diff
@@ -781,6 +783,8 @@
     <string name="notification_channel_playing_description">Allows to control playback. This is the main notification you see while playing a podcast.</string>
     <string name="notification_channel_error">Errors</string>
     <string name="notification_channel_error_description">Shown if something went wrong, for example if download or gpodder sync fails.</string>
+    <string name="notification_channel_auto_download">Auto Downloads</string>
+    <string name="notification_channel_error_auto_download">Shown when episodes are automatically downloaded.</string>
```

Comment:
Maybe "have been" downloaded? Not a native English speaker but the notification is shown after the downloads were completed. "Are" sounds like the download is still running.

Reply:
Changed.